qiskit/tools/monitor/job_monitor.py

```diff
@@ -52,19 +57,22 @@ def _text_checker(job, interval, _interval_set=False):
         elif len(msg) > msg_len:
             msg_len = len(msg)
 
-        if msg != prev_msg:
-            print('\r%s: %s' % ('Job Status', msg), end='')
+        if msg != prev_msg and not quiet:
+            print('\r%s: %s' % ('Job Status', msg), end='', file=_outstream)
             prev_msg = msg
-    print('')
+    if not quiet:
+        print('', file=_outstream)
 
 
-def job_monitor(job, interval=None, monitor_async=False):
+def job_monitor(job, interval=None, monitor_async=False, quiet=False, to_file=None):
```

Comment:
```suggestion
def job_monitor(job, interval=None, monitor_async=False, quiet=False, output=sys.stdout):
```

Reply:
Will make these changes today-ish, tomorrow-ish ...

Reply:
And yes, @mtreinish , I misunderstood the Github workflow and made the same pr-close-open mistake on a couple of submissions. I _think_ I have it straight now :)